app/mailers/onboarding_mailer.rb

```diff
@@ -47,7 +53,7 @@ def data_enabled_email
   def welcome_email
     @school = params[:school]
     @title = @school.name
-    @to = user_emails(params[:users])
-    make_bootstrap_mail(to: @to)
+    @user = params[:user]
```

Comment:
Changes to parameter mean the preview for this is broken, its still using `users`